### ISSUE TITLE
Hide undefined Braille Opcodes

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2949]
**Reported by quetzatl on 2013-01-26 17:26**
NVDA should be able to hide the braille opcodes which are not defined in the braille tables
**Blocking #5091**

**Comment by josephsl** [comment]
Hi,

Nine years later...

When I press current character command three times, character code is shown in braille. If this is the behavior being sought, then I vote to close this issue.

Thanks.

**Comment by josephsl** [comment]
Hi,

Actually, on second thought, this issue should be closed as the use case isn't apparent from initial comments (started in 2013 with latest comments from 2015). If people need braille output to anounce character descriptions, then it would be better to suggest it via a new issue.

Thanks.
